app/controllers/follows.rb

```diff
@@ -0,0 +1,10 @@
+# NOT WORKING YET
+
+get '/followers/new' do
```

Comment:
This route controller is not necessary cause you would only want to create a follow not present them with a form. Think about how Twitter works.